lldb/test/API/commands/settings/use_source_cache/TestUseSourceCache.py

```diff
@@ -18,7 +18,7 @@ def test_set_use_source_cache_false(self):
         self.set_use_source_cache_and_test(False)
 
     @skipIf(hostoslist=no_match(["windows"]))
-    @skipIf(oslist=["windows"])  # Fails on windows 11
+    @expectedFailureAll(oslist=["windows"])
```

Comment:
Looking at the tested functionality (and the surrounding code), this should probably be `hostoslist=windows`

Reply:
Re: @DavidSpickett comment below. I am using the lldb-aarch64-windows builder as an example:

- Run in cmd.exe
- LLDB_TEST_COMPILER is the newly built clang-cl (top-level build uses host's MSVC)
- `--skip-test-category=watchpoint`
- LLDB_ENFORCE_STRICT_TEST_REQUIREMENTS=ON
- Python 3.9

My host system:
- Win 11 Home Version 10.0.22631 Build 22631
- Dell XPS 8960

I haven't explicitly disabled hardware code breakpoints. What is the default and how do I do that?

Reply:
To reproduce my local build if you have access to a machine and a clean checkout llvm/llvm-project: Run this in a developer command prompt in the root of the checkout:

```
set MSVC_INSTALL_DIR=C:\Program Files\Microsoft Visual Studio\2022\Community
set "PATH=%MSVC_INSTALL_DIR%\DIA SDK\bin\amd64;%PATH%"

cmake -G Ninja ^
      -S llvm ^
      -B build ^
      -D Python3_EXECUTABLE="C:\Program Files\Python312\python.exe" ^
      -D CMAKE_C_COMPILER="cl" ^
      -D CMAKE_CXX_COMPILER="cl" ^
      -D CMAKE_C_COMPILER_LAUNCHER=ccache ^
      -D CMAKE_CXX_COMPILER_LAUNCHER=ccache ^
      -D CMAKE_BUILD_TYPE="Release" ^
      -D CMAKE_LINKER="C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\Llvm\x64\bin\lld-link.exe" ^
      -D LLVM_ENABLE_PROJECTS="clang;lldb;lld" ^
      -D LLVM_HOST_TRIPLE="x86_64-unknown-windows-msvc" ^
      -D LLVM_ENABLE_LLD="YES" ^
      -D LLVM_ENABLE_LIBEDIT="NO" ^
      -D LLVM_ENABLE_DIA_SDK="OFF" ^
      -D LLDB_ENABLE_PYTHON=1 ^
      -D LLDB_PYTHON_EXE_RELATIVE_PATH="python.exe" ^
      -D LLDB_TEST_USER_ARGS=--skip-category=watchpoint ^
      -D LLDB_TEST_COMPILER="clang-cl" ^
      -D LLDB_ENFORCE_STRICT_TEST_REQUIREMENTS=1 ^
      -D LLDB_USE_LLDB_SERVER=1 ^
      -D CMAKE_CXX_STANDARD=14
```

Reply:
I made the following changes and I'm seeing far fewer failing tests now:
- `LLDB_TEST_COMPILER` - leave this unset
- `CMAKE_C_COMPILER="C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\Llvm\x64\bin\clang-cl.exe"`
- `CMAKE_CXX_COMPILER="C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\Llvm\x64\bin\clang-cl.exe"`

In short the compilers I was using caused many of the tests to fail.  I am only seeing around 17 failures now in a clean checkout and will fix this one up. Thanks for the course correction!